packages/elements/src/components/ui/collapsible/Collapsible.module.css

```diff
@@ -0,0 +1,107 @@
+.collapsible {
+  --swui-collapsible-animation-time: var(--swui-animation-time-medium);
+  --swui-collapsible-header-background-color: #fff;
```

Comment:
All of these colors should be in default-theme.
Wait until Checkbox och RadioButtons have been merged, that branch adds a lot more colors.

Reply:
Also in button 😃